ports/servoshell/keyutils.rs

```diff
@@ -20,100 +20,339 @@ pub const CMD_OR_ALT: Modifiers = Modifiers::META;
+        LogicalKey::Character(string) => return Key::Character(string.to_string()),
+        LogicalKey::Unidentified(_) => return Key::Unidentified,
+        LogicalKey::Dead(_) => return Key::Unidentified,
+    };
+    // TODO: map the remaining keys (F25..=F35) if possible
+    match named_key {
         // printable: Key1 to Key0
         // printable: A to Z
-        LogicalKey::Named(NamedKey::Escape) => Key::Escape,
-        LogicalKey::Named(NamedKey::F1) => Key::F1,
-        LogicalKey::Named(NamedKey::F2) => Key::F2,
-        LogicalKey::Named(NamedKey::F3) => Key::F3,
-        LogicalKey::Named(NamedKey::F4) => Key::F4,
-        LogicalKey::Named(NamedKey::F5) => Key::F5,
-        LogicalKey::Named(NamedKey::F6) => Key::F6,
-        LogicalKey::Named(NamedKey::F7) => Key::F7,
-        LogicalKey::Named(NamedKey::F8) => Key::F8,
-        LogicalKey::Named(NamedKey::F9) => Key::F9,
-        LogicalKey::Named(NamedKey::F10) => Key::F10,
-        LogicalKey::Named(NamedKey::F11) => Key::F11,
-        LogicalKey::Named(NamedKey::F12) => Key::F12,
-        // F13 to F15 are not mapped
-        LogicalKey::Named(NamedKey::PrintScreen) => Key::PrintScreen,
-        // Scroll not mapped
-        LogicalKey::Named(NamedKey::Pause) => Key::Pause,
-        LogicalKey::Named(NamedKey::Insert) => Key::Insert,
-        LogicalKey::Named(NamedKey::Home) => Key::Home,
-        LogicalKey::Named(NamedKey::Delete) => Key::Delete,
-        LogicalKey::Named(NamedKey::End) => Key::End,
-        LogicalKey::Named(NamedKey::PageDown) => Key::PageDown,
-        LogicalKey::Named(NamedKey::PageUp) => Key::PageUp,
-        LogicalKey::Named(NamedKey::ArrowLeft) => Key::ArrowLeft,
-        LogicalKey::Named(NamedKey::ArrowUp) => Key::ArrowUp,
-        LogicalKey::Named(NamedKey::ArrowRight) => Key::ArrowRight,
-        LogicalKey::Named(NamedKey::ArrowDown) => Key::ArrowDown,
-        LogicalKey::Named(NamedKey::Backspace) => Key::Backspace,
-        LogicalKey::Named(NamedKey::Enter) => Key::Enter,
+        NamedKey::Escape => Key::Escape,
+        NamedKey::F1 => Key::F1,
+        NamedKey::F2 => Key::F2,
+        NamedKey::F3 => Key::F3,
+        NamedKey::F4 => Key::F4,
+        NamedKey::F5 => Key::F5,
+        NamedKey::F6 => Key::F6,
+        NamedKey::F7 => Key::F7,
+        NamedKey::F8 => Key::F8,
+        NamedKey::F9 => Key::F9,
+        NamedKey::F10 => Key::F10,
+        NamedKey::F11 => Key::F11,
+        NamedKey::F12 => Key::F12,
+        NamedKey::F13 => Key::F13,
+        NamedKey::F14 => Key::F14,
+        NamedKey::F15 => Key::F15,
+        NamedKey::F16 => Key::F16,
+        NamedKey::F17 => Key::F17,
+        NamedKey::F18 => Key::F18,
+        NamedKey::F19 => Key::F19,
+        NamedKey::F20 => Key::F20,
+        NamedKey::F21 => Key::F21,
+        NamedKey::F22 => Key::F22,
+        NamedKey::F23 => Key::F23,
+        NamedKey::F24 => Key::F24,
+        NamedKey::F25 => Key::Unidentified,
```

Comment:
These appear to have [been added in the latest release of keyboard-types](https://github.com/pyfisch/keyboard-types/commit/d3b99418886256dd46bc091715ef300f6dae034c) if you want to update that while you're at it.

Reply:
F25..=F35 are now supported.